addons/web/static/src/core/l10n/localization.js

```diff
@@ -7,6 +7,15 @@ const notReadyError = new Error(
     "Localization parameters not ready yet. Maybe add 'localization' to your dependencies?"
 );
 
+/**
+ * This is the main object holding user specific data about the localization. Its basically
+ * the JS counterpart of the "res.lang" model.
+ * It is useful to directly access those data anywhere, even outside Components.
+ * Important Note: its data are actually loaded by the localization_service,
+ *   so a code like the following would not work:
+ * ` import { localization } from "@web etc...";
+ *   const dateFormat = localization.dateFormat;`
```

Comment:
it's actually how you're supposed to use it, isn't it?

Reply:
not really, we are supposed to import localization, but only use its members in Components/service functions, right ?

Reply:
Of course, I thought you were saying we can't import the localization object, but we have to go through the service.